wardrobe/kitties/src/lib.rs

```diff
@@ -199,6 +242,7 @@ impl Default for KittyData {
             free_breedings: 2,
             dna: KittyDNA(H256::from_slice(b"mom_kitty_1asdfasdfasdfasdfasdfa")),
             num_breedings: 3,
+            name: *b"kity",
```

Comment:
🤔

Reply:
I don't see any comment here, Please let me know if I need to change anything.
Basically I choose the default name based on my intuition. Suggest if you have a better name :). Thanks in advance.

Reply:
Sorry for being unclear! If the name is not limited to 4, then it's just a typo.

Reply:
I intend to limit the name of the kitty to 4 chars only. This limitation is to reduce the storage space in on-chain.
Spo this spelling "kity" is intended only. Thanks.